crates/corecrypto/src/hashes.rs

```diff
@@ -0,0 +1,21 @@
+use sha2::Digest;
+use sha2::Sha256;
+use sha3::Keccak256;
+
+pub fn sha256(data: &[u8]) -> [u8; 32] {
+    let mut hasher = Sha256::new();
+    hasher.update(data);
+    let result = hasher.finalize();
+    let mut digest = [0u8; 32];
+    digest.copy_from_slice(&result);
+    digest
+}
```

Comment:
We actually don't need this exposed in swift because the built-in CryptoKit stuff provides a `SHA256` function, unless there's something special about how we're doing it here.

Reply:
Nope, nothing special in this sha256! I can remove the function.